src/main/java/TaskList.java

```diff
@@ -0,0 +1,84 @@
+import java.io.IOException;
+
+public class TaskList {
+    Task[] taskList = new Task[100];
+    // The first index that is empty
+    private int firstEmptyIndex = 0;
+
+    public void addTask(String input, Task.TaskType taskType, boolean isDone) throws IOException {
+        Task task = taskType == Task.TaskType.ToDo
+                ? new Todo(input, isDone)
+                : taskType == Task.TaskType.Event
+                ? new Event(input, isDone)
+                : new Deadline(input, isDone);
```

Comment:
This consecutive ternary operators can be confusing to other coders, might be easier to read if you were to use switch or if else statements directly.